src/core/jsonschema/frame.cc

```diff
@@ -1,10 +1,20 @@
+#include <sourcemeta/core/json_value.h>
```

Comment:
Let's make modules only include public headers of other modules

```suggestion
#include <sourcemeta/core/json.h>
```

The idea is that only `src/json` can be internally aware that there is a `json_value.h` header, and every consumer should just require the public one